react-big-calendar/react-big-calendar-tests.tsx

```diff
@@ -1,7 +1,7 @@
 import * as React from "react";
 import * as ReactDOM from "react-dom";
 import * as ReactDOMServer from "react-dom/server";
-import BigCalendar from 'react-big-calendar';
+import * as BigCalendar from "react-big-calendar";
```

Comment:
@frankwallis please change to either:
- `import BigCalendar = require('react-big-calendar)`
or
- import BigCalendar from 'react-big-calendar' with tsconfig --allowSyntheticDefaultImports set to true